### PR TITLE
Update oma-dm-protocol-support.md

### PR DESCRIPTION
**Pull request by nenonix**
Grammar and formatting fixes https://github.com/MicrosoftDocs/windows-itpro-docs/issues/673

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/client-management/mdm/oma-dm-protocol-support.md [hunk 1]
```diff
@@ -314,13 +314,13 @@ For more information about Basic or MD5 client authentication, MD5 server authen
 
 ## User targeted vs. Device targeted configuration
 
-For CSPs and policies that supports per user configuration, MDM server could send user targeted setting values to the device the user that enrolled MDM is actively logged in. The device notifies the server the login status via a device alert (1224) with Alert type = in DM pkg\#1.
+For CSPs and policies that support per user configuration, MDM server could send user targeted setting values to the device the user that enrolled MDM is actively logged in. The device notifies the server of the login status via a device alert (1224) with Alert type = in DM pkg\#1.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/oma-dm-protocol-support.md]: For CSPs and policies that support per user configuration, the MDM server could send user targeted setting values to the device that the user who enrolled MDM is actively logged into. The device notifies the server of the login status via a device alert (1224) with Alert type = in DM pkg\#1.

Comment (mypil) [reply windows/client-management/mdm/oma-dm-protocol-support.md]: @nenonix - can you possibly make the necessary suggested changes made by Johan or merge the new commit for this PR? Thanks.

**Comment by mypil** [comment]
@kenwith - this article is authored by @MariciaAlforque. Would you be able to merge the content for this PR based on issue #673?

Thank you.

File: windows/client-management/mdm/oma-dm-protocol-support.md [hunk 2]
```diff
@@ -314,7 +314,7 @@ For more information about Basic or MD5 client authentication, MD5 server authen
 
 ## User targeted vs. Device targeted configuration
 
-For CSPs and policies that support per user configuration, MDM server could send user targeted setting values to the device the user that enrolled MDM is actively logged in. The device notifies the server of the login status via a device alert (1224) with Alert type = in DM pkg\#1.
+For CSPs and policies that support per user configuration, the MDM server can send user targeted setting values to the device that a user, who has enrolled with MDM, is actively logged into. The device notifies the server of the login status via a device alert (1224) with Alert type = in DM pkg\#1.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/oma-dm-protocol-support.md]: Copy edit - no changes.

File: windows/client-management/mdm/oma-dm-protocol-support.md [hunk 3]
```diff
@@ -314,7 +314,7 @@ For more information about Basic or MD5 client authentication, MD5 server authen
 
 ## User targeted vs. Device targeted configuration
 
-For CSPs and policies that support per user configuration, the MDM server can send user targeted setting values to the device that a user, who has enrolled with MDM, is actively logged into. The device notifies the server of the login status via a device alert (1224) with Alert type = in DM pkg\#1.
+For CSPs and policies that support per user configuration, the MDM server can send user targeted setting values to the device that a MDM-enrolled user is actively logged into. The device notifies the server of the login status via a device alert (1224) with Alert type = in DM pkg\#1.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/oma-dm-protocol-support.md]: For CSPs and policies that support per user configuration, the MDM server can send user targeted setting values to the device that **an** MDM-enrolled user is actively logged into. The device notifies the server of the login status via a device alert (1224) with Alert type = in DM pkg\#1.

File: windows/client-management/mdm/oma-dm-protocol-support.md [hunk 4]
```diff
@@ -314,13 +314,13 @@ For more information about Basic or MD5 client authentication, MD5 server authen
...
 
 The data part of this alert could be one of following strings:
 
--   user – the user that enrolled the device is actively login. The MDM server could send user specific configuration for CSPs/policies that support per user configuration
+-   user – the user that enrolled the device is actively logged in. The MDM server could send user specific configuration for CSPs/policies that support per user configuration
 -   others – another user login but that user does not have an MDM account. The server can only apply device wide configuration, e.g. configuration applies to all users in the device.
--   none – no active user login. The server can only apply device wide configuration and available configuration is restricted to the device environment (no active user login
+-   none – no active user login. The server can only apply device wide configuration and available configuration is restricted to the device environment (no active user login).
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/oma-dm-protocol-support.md]: -   none – no active user login. The server can only apply **device-wide** configuration and available configuration is restricted to the device environment (no active user login).

File: windows/client-management/mdm/oma-dm-protocol-support.md [hunk 5]
```diff
@@ -314,13 +314,13 @@ For more information about Basic or MD5 client authentication, MD5 server authen
 
 ## User targeted vs. Device targeted configuration
 
-For CSPs and policies that supports per user configuration, MDM server could send user targeted setting values to the device the user that enrolled MDM is actively logged in. The device notifies the server the login status via a device alert (1224) with Alert type = in DM pkg\#1.
+For CSPs and policies that support per user configuration, the MDM server can send user targeted setting values to the device that a MDM-enrolled user is actively logged into. The device notifies the server of the login status via a device alert (1224) with Alert type = in DM pkg\#1.
 
 The data part of this alert could be one of following strings:
 
--   user – the user that enrolled the device is actively login. The MDM server could send user specific configuration for CSPs/policies that support per user configuration
+-   user – the user that enrolled the device is actively logged in. The MDM server could send user specific configuration for CSPs/policies that support per user configuration
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/oma-dm-protocol-support.md]: -   user – the user that enrolled the device is actively logged in. The MDM server could send **user-specific** configuration for CSPs/policies that support per user configuration

**Comment by illfated** [comment]
@JohanFreelancer9 : the Pull Request above has already been merged, so your belated requests for change needs to be moved to a new PR or issue ticket (depending on your preferred route of action).

**Comment by mypil** [comment]
@nenonix - were you able to merge the new commits on the copy/edit review done by @JohanFreelancer9 before jdeckerms merged the entire PR content?  if not, there may be a need to create another PR with the copy/edit recommendations. thanks

**Comment by nenonix** [comment]
No need, I incorporated those that were valid,

**Pull request merged**